paddle/fluid/operators/fused/fusion_conv_inception_op.cu

```diff
@@ -61,7 +61,7 @@ class CUDNNConvInceptionFusionOpKernel : public framework::OpKernel<T> {
     T* temp_data = temp_outs[0]->mutable_data<T>(input->dims(), ctx.GetPlace());
 
     DataLayout layout = DataLayout::kNCHW;
-    std::vector<int> in_dim = framework::vectorize2int(input->dims());
+    std::vector<int> in_dim = framework::vectorize<int>(input->dims());
```

Comment:
I'm not sure if You want to dive so much into changes but some lines might be improved by using `auto` keyword

Reply:
I use one `sed` command to do this change, so I don't use `auto` keyword.